cmd/appsctl/openfaas.go

```diff
@@ -0,0 +1,128 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/mattermost-plugin-apps/apps"
+	"github.com/mattermost/mattermost-plugin-apps/upstream/upopenfaas"
+)
+
+func init() {
+	rootCmd.AddCommand(openfaasCmd)
+
+	// provision
+	openfaasCmd.AddCommand(openfaasProvisionCmd)
+	openfaasProvisionCmd.Flags().BoolVar(&shouldUpdate, "update", false, "Update functions if they already exist. Use with caution in production.")
+	openfaasProvisionCmd.Flags().BoolVar(&install, "install", false, "Install the deployed App to Mattermost")
+	openfaasProvisionCmd.Flags().StringVar(&dockerRegistry, "docker-registry", "", "Docker image prefix, usually the docker registry to use for deploying functions.")
+
+	// test
+	// openfaasCmd.AddCommand(openfaasTestCmd)
```

Comment:
Why is the command commented out?

Reply:
Not implemented yet. The next "big" step for `appsctl` is to be integrated into `/apps` command, as @jwilander suggested at the R&D demo. The entire "test" functionality can then migrate to the plugin, where it belongs.